jsonrpc.py

```diff
@@ -0,0 +1,149 @@
+# SPDX-License-Identifier: GPL-3.0
+import io
+import json
+import os
+import re
+
+import xbmc
+
+
+class JsonRpcApi(object):
+    def __init__(self):
+        """ Initialise the JSON RPC API Implementation. """
+        self._paths = xbmc.get_add_on_info_from_calling_script()
+
+
+class Addons(JsonRpcApi):
```

Comment:
I prefer to not include any classes that are not used. So don't include all of these.

Reply:
:+1: